README.md

```diff
@@ -31,16 +31,44 @@ Clipboard Utilities
  - Windows - clip
  - Linux   - xsel
 
+Commands
+--------
+
+System copy adds vim commands for copying `:SystemCopy`,
+`:SystemCopyLine` as well as for pasting `:SystemPaste`.
```

Comment:
Can you explain the value of these as commands? This plugin mostly makes sense to me via the mappings.